### ISSUE TITLE
Python 3.10 support

### ISSUE THREAD

**Issue by sr-verde** [OnroerendErfgoed/language-tags#87]
Currently, this version does not officially support Python 3.10. I think there is no reason not to announce support for this Python version, isn't it? At least for me it works well with Python 3.10.

:v:

**Comment by koenedaele** [comment]
True. Next will officially support 3.10, 3.9 and 3.8. We'll drop support for 3.6 and 3.7
